modules/standard/Time.chpl

```diff
@@ -623,7 +639,7 @@ enum Day       { sunday=0, monday, tuesday, wednesday, thursday, friday, saturda
    */
   @unstable "tzinfo is unstable; its type may change in the future"
   proc time.init(hour=0, minute=0, second=0, microsecond=0,
-                 in tzinfo: shared TZInfo?) {
+                 in tzinfo: shared Timezone?) {
```

Comment:
It might be nice to also change the argument name to `tz` to match some of the other methods that take a `Timezone` and to get rid of the tzinfo naming.

Reply:
There are actually a bunch of functions with `tzinfo` arguments. I opened https://github.com/chapel-lang/chapel/issues/20870 to ask for this renaming.